include/boost/math/fft/real_algorithms.hpp

```diff
@@ -155,11 +155,19 @@
           *vx = prev_ux - cos * prev_vy - sin * prev_uy;
           *vy =-prev_vx + cos * prev_uy - sin * prev_vy;
         }
+        //if(prev_len>=2)
+        //{
+        //  const int j = prev_len/2;
+        //  T* u = out + i + j, *v = out + i + j + prev_len;
+        //  T Bu = *u, Bv = *v;
+        //  *u = Bu;
+        //  *v = Bv;
+        //}
```

Comment:
these lines are real code but the transformation is the identity, so there is no harm in commenting them.